dap_aria_mapping/pipeline/data_collection/requirements_openalex_flow.txt

```diff
@@ -0,0 +1,6 @@
+boto3
```

Comment:
this is just for batch, as the python env used on batch isn't the same as the local env